browser/js/w2ui-1.5.rc1.js

```diff
@@ -2529,7 +2529,7 @@ w2utils.event = {
             if (typeof options.onRender === 'function' && typeof options.render !== 'function') options.render = options.onRender;
             // since only one overlay can exist at a time
             $.fn.w2menuClick = function (event, index) {
-                var keepOpen = false;
+                var keepOpen = options.keepOpen || false;
```

Comment:
Is this intentional? Looks like a debugging leftover.

Reply:
Yes, I needed to touch the w2ui library file as agreed with @eszkadev and changed this. Not sure why this was hardcoded to false however I did not want to break the existing code and want to pass this option inside the options object so I made it this way to be safe. Since we dont compile w2ui from npm I guess this is alright.

Reply:
Ah, I see, thanks. Perhaps it is worth to add a comment above to line to explain why is this necessary? So a next person does not "clean it up" for you. :-)